R/json.R

```diff
@@ -40,3 +40,11 @@ to_upper_first <- function(x) {
   substr(x, 1, 1) <- toupper(substr(x, 1, 1))
   x
 }
+
+## Read json and apply a class so we can use method dispatching for json
+hintr_geojson_read <- function(path) {
+  json <- geojsonio::geojson_read(path, method = "local")
```

Comment:
Is this correct? Does `geojsonio::geojson_read` not return an R object? In which case pretending that it's json must do bad things if we return that anywhere.  Looking here specifically at adding the `json` class.

Reply:
It just returns a list as far as I can see. I was thinking about that class on the way in - perhaps more appropriate to wrap in a `geojson` class as the assert method which uses it for method dispatch the behaviour specific to `geojson`. Either that or I we can just use a `switch` on the file type instead.

Reply:
Something like `geojson` would be better because `json` is the class we use in `json_verbatim`

Reply:
Fixed this